sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/CombineWithContext.java

```diff
@@ -28,6 +29,7 @@
  *
  * <p>{@link CombineFnWithContext} is for users to extend.
  */
+@Internal
```

Comment:
I don't believe this is meant to be marked internal

Reply:
ok removing it